11:59:14.869 INFO Test execution started: second_test
11:59:14.870 INFO Browser: chrome-remote
11:59:14.870 INFO Environment: test_env
11:59:14.870 INFO Using data:
    order_type: Plats
    document_type: CORRECTION OF PLAT (PL)
    num_of_pages: 1
    num_of_lots: 1
    section: GRANTOR
    first_name: First Name
    alert_text: Please Enter Account OR Email OR Name
    url: http://{}:{}@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer=
11:59:15.548 ERROR TypeError: an integer is required (got type str)
Traceback (most recent call last):
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\test_runner\test_runner.py", line 173, in run_test
    self.test_module.test(execution.data)
  File "C:\Users\alexandr.may\testdir\projects\some_project\tests\second_test.py", line 32, in test
    file2 = open('C:/Users/alexandr.may/testdir/projects/some_project/temp.json', 'r', 'utf-8')
TypeError: an integer is required (got type str)

11:59:15.715 INFO Test Result: CODE ERROR
